src/components/FormStepSummary/index.js

```diff
@@ -42,7 +42,9 @@ const FormStepSummary = ({slug, name, data, editStepText}) => {
   return (
     <div className={getBEMClassName('summary')}>
       <div className={getBEMClassName('summary__step-header')}>
-        <div className={getBEMClassName('summary__step-name')}>{name}</div>
+        <div className={getBEMClassName('summary__step-name')}>
+          <h3>{name}</h3>
+        </div>
```

Comment:
```suggestion
        <h3 className={getBEMClassName('summary__step-name')}>{name}</h3>
```
is there a reason we can't do this?

We really don't want to rely on global/browser-default styles for UI elements and every component needs to be explicitly styled via a BEM class name. Haven't checked how this looks, but this should look the same and be semantic, no?

Reply:
Yes, I think still works for accessibility - I might still need to set margins and font sizes.